packages/svelte/src/reactivity/set.js

```diff
@@ -150,4 +151,7 @@ export class ReactiveSet extends Set {
 	get size() {
 		return get(this.#size);
 	}
+	[INSPECT_SYMBOL]() {
```

Comment:
We should probably add these to the prototype only in DEV to reduce bundle size.